better-restaurant/src/components/SigninComponent.js

```diff
@@ -2,13 +2,108 @@ import React from 'react';
 import styled from 'styled-components'
 
 const Container = styled.div`
-  
+  padding-left: 20px;
 `
+//figma.com
+
+const LoginTitle = styled.div`
+	margin-height: 25vh;
+	font-family: Inter;
+	font-style: normal;
+	font-weight: bold;
+	font-size: 36px;
+	line-height: 22px;
+	padding-left: 22vw;
+
+	color: #1A61BD;
+`
+
+const MainBox = styled.div`
+	width: 72vw;
+	height: 76vh;
+	background: #FFFFFF;
+	border: 3px solid #E9E9EF;
+	box-sizing: border-box;
+	box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+`
+
+const Index = styled.div`
+	width: 102px;
+	height: 25px;
+	font-size: 20px;
+	line-height: 22px;
+
+	display: flex;
+	align-items: center;
+
+	color: #000000;
+`
+const Input = styled.div`
+	width: 47vw;
+	height: 64px;
+
+	background: #E5E5E5;
+	border-radius: 10px;
+`
+
+
+
+const GrayIndex3 = styled.div`
+	width: 126px;
+	height: 34px;
+	font-size: 20px;
+	line-height: 22px;
+
+	color: #85859A;
+`
+const SignUpText = styled.div`
+	font-size: 25px;
+
+	display: flex;
+	align-items: center;
+	text-align: center;
+
+	color: #4286F5;
+`
+const BlueIndex2 = styled.div`
+	width: 47vw;
+	height: 8vh;
+
+	background: #4286F5;
+	border: 1px solid #437DD1;
+	box-sizing: border-box;
+	border-radius: 10px;
+`
+
+const WhiteIndex = styled.div`
+	font-weight: bold;
+	font-size: 36px;
+
+	color: #FFFFFF;
+`
+
+const Checkbox = styled.input`
+	background-color: blue;
+	color: blue;
+`
+
 
 export default function SigninComponent(){
   return (
     <Container>
-      Signin
+			<MainBox>
+				<LoginTitle>부대계정로그인</LoginTitle>
+				<Index>ID</Index>
+				<Input></Input>
+				<Index>PASSWORD</Index>
+				<Input></Input>
+				<Checkbox type="checkbox"/>
+				<GrayIndex3>자동 로그인</GrayIndex3>
+				<SignUpText>회원가입</SignUpText>
+				<BlueIndex2>
+					<WhiteIndex>로그인</WhiteIndex>
+				</BlueIndex2>
```

Comment:
위와 비슷함.
근데 "박스 내부의 텍스트"라면 굳이 박스의 텍스트를 다른 div로 만들 필요가 없음.
css의 color값은 내부 text의 색깔만 바꿔주므로, css로 상위 박스에서 글자를 가운데로 모아만 준다면, 굳이 내부 text의 구역을 따로 만들 필요가 없음.
```suggestion
const LoginButton = styled.button`
  width: 0; //박스에 맞게
  height: 0; // 박스에 맞게
  text-align: center;
  vertical-align: center;
  color: white;
  background-color: blue;
  `
  
... 
<LoginButton>
  로그인
</LoginButton>
```